src/common/style/icon_color_css.ts

```diff
@@ -0,0 +1,43 @@
+import { css } from "lit-element";
+
+export const iconColorCSS = css`
+  ha-icon[data-domain="alarm_control_panel"][data-state="disarmed"],
+  ha-icon[data-domain="alert"][data-state="on"],
+  ha-icon[data-domain="automation"][data-state="on"],
+  ha-icon[data-domain="binary_sensor"][data-state="on"],
+  ha-icon[data-domain="calendar"][data-state="on"],
+  ha-icon[data-domain="camera"][data-state="streaming"],
+  ha-icon[data-domain="cover"][data-state="open"],
+  ha-icon[data-domain="fan"][data-state="on"],
+  ha-icon[data-domain="light"][data-state="on"],
+  ha-icon[data-domain="input_boolean"][data-state="on"],
+  ha-icon[data-domain="lock"][data-state="unlocked"],
+  ha-icon[data-domain="media_player"][data-state="paused"],
+  ha-icon[data-domain="media_player"][data-state="playing"],
+  ha-icon[data-domain="script"][data-state="running"],
+  ha-icon[data-domain="sun"][data-state="above_horizon"],
+  ha-icon[data-domain="switch"][data-state="on"],
+  ha-icon[data-domain="timer"][data-state="active"],
+  ha-icon[data-domain="timer"][data-state="paused"],
```

Comment:
I would not consider paused to light up.

Reply:
I thought about putting this and media_player paused into their own have having them pulse, but didn't think many would like that

Reply:
![Peek 2020-01-22 12-50](https://user-images.githubusercontent.com/1287159/72924139-0757f080-3d16-11ea-887b-e9ee53da6cb3.gif)

Reply:
Nah, that's a lot of attention grabbing. I suggest we keep it uncolored for now.